### PR TITLE
Minor AuthClient Helper & -Service Cleanups 

### LINKED ISSUE #1: New Posts below Sticky Items
New posts appears below sticky posts

### PR DESCRIPTION
**Pull request by yuriimaz**
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `develop` branch, _not_ the `master` branch if no hotfix
- [x] When resolving a specific issue, it's referenced in the PR's description (e.g. `Fix #xxx[,#xxx]`, where "xxx" is the Github issue number)
- [ ] All tests are passing
- [ ] New/updated tests are included
- [ ] Changelog was modified

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**
Changes and corrections are necessary for correct authorization using the auth-linkedin module if the user is not yet registered.

### CODE REVIEW

File: protected/humhub/modules/user/services/AuthClientService.php [hunk 1]
```diff
@@ -120,6 +119,16 @@ public function createRegistration(): ?Registration
             return null;
         }
 
+        if (
+            $this->authClient->getId() === 'linkedin' &&
```
Comment (luke-) [inline protected/humhub/modules/user/services/AuthClientService.php:123]: It should not be necessary to include module specific logic in the core.

Maybe you can use this example: to map the attributes on Module level.
https://github.com/humhub-contrib/auth-github/blob/master/authclient/GithubAuth.php#L30

**Review by luke- — CHANGES_REQUESTED** [review]

File: protected/humhub/modules/user/services/AuthClientService.php (hunk 1 above)
Comment (yuriimaz) [reply protected/humhub/modules/user/services/AuthClientService.php:123]: Ok

Comment (yuriimaz) [reply protected/humhub/modules/user/services/AuthClientService.php:123]: But you have some problem with 'lastname' parameters. Look carefully PR

Comment (luke-) [reply protected/humhub/modules/user/services/AuthClientService.php:123]: @yuriimaz Yes thanks, I think we should merge this part.

**Review by luke- — APPROVED** [review]

**Pull request merged**